js/projects.js

```diff
@@ -0,0 +1,127 @@
+import projectInfo from './projectInfo.js';
+
+document.querySelector('#works').innerHTML = projectInfo.map(
+  (info) => `
+        <!--card  -->
+        <div class="card">
+          <img
+            class="card__image"
+            src=${info.featuredImage}
+            alt="Image for the tonic project indicating 'From high school dropout to product developer at Facebook and Lyft: Tanner Christensen'"
+          />
+          <img
+            class="card__image-desktop"
+            src="images/project1_desktop.png"
+            alt="Desktop version image for Nature project indicating 'Catchy look inspiring bionic forms $35'"
+          />
+          <div class="card__body">
+            <h2 class="card__heading">${info.name}</h2>
+            <div class="card__details">
+              <h4>${info.author}</h4>
+              <div>
+                <ul class="card__list">
+                  <li class="card__role">${info.stack}</li>
+                  <li class="card__year">${info.year}</li>
+                </ul>
+              </div>
+            </div>
+            <p class="card__text">
+             ${info.description}
+            </p>
+            <div class="card__languages">
+              <button class="btn--primary">${info.technologies[0]}</button>
+              <button class="btn--primary">${info.technologies[1]}</button>
+              <button class="btn--primary">${info.technologies[2]}</button>
+            </div>
+            <div class="card__btn-wrapper">
+            <button type="button" class="btn--secondary card__btn">See Project</button>
+            </div>
+          </div>
+        </div>
+        `,
+)
+  .join('');
+
+for (let i = 0; i <= projectInfo.length; i += 1) {
+  if (i % 2 === 1) {
+    document.querySelectorAll('.card')[i].classList.add('order');
+  }
+}
+
+//   Popup
+document.querySelector('#popups').innerHTML = projectInfo.map(
+  (info) => `
+    <div class="popup-container">
```

Comment:
- [Optional] Dear @mutugi-c, Your pop-up is looking good already, and I like it. But, It seems for most of the pop-ups I have to scroll to see the whole section. As you can notice from the Figma design, this is not true. The ability to let users have the whole pop-up on a single page has a UX benefit and I suggest you address it. 

https://user-images.githubusercontent.com/69077061/223785851-6daf377a-0e50-46f4-b28f-ad150243d75d.mp4

Reply:
Hello @TadesseDev. Great point. I'll surely address it. Thanks for the hint.